go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -1435,15 +1436,15 @@ func (eIPC *egressIPClusterController) validateEgressIPStatus(name string, items
 			isOVNManaged, err := util.IsOVNManagedNetwork(node, ip)
 			if err != nil {
 				klog.Errorf("Allocator error: failed to determine if IP %s is part of an OVN managed network for "+
-					"egress IP %s: %v", ip.String(), name, err)
+					"egress IP %s: %v", eIPStatus.EgressIP, name, err)
```

Comment:
If `eIPStatus.EgressIP` can be an empty string, you could use `%#v` instead of `%s` so that an emtpty string has better visibility
```
Allocator error: failed to determine if IP  is part of an OVN managed network for egress IP <name>: <error>
```
vs
```
Allocator error: failed to determine if IP "" is part of an OVN managed network for egress IP <name>: <error>
```

Reply:
good point - it can be "". I switched to using `%q` though instead of `%#v`. I think more folks understand this format specifier.

Reply:
It cannot be blank but I think this is a good idea anyway.